src/store/audio.ts

```diff
@@ -1212,6 +1212,12 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
 
   DEFAULT_PROJECT_FILE_BASE_NAME: {
     getter: (state) => {
+      // NOTE: 起動時にソングエディタが開かれると、トークの初期化が行われずAudioCellが作成されないので、
+      //       state.audioKeys.length === 0になる
```

Comment:
あ、ちょっとここ変えさせていただきます！

```suggestion
      // NOTE: 起動時にソングエディタが開かれた場合、トークの初期化が行われずAudioCellが作成されない
      // TODO: ソングエディタが開かれてい場合はこの関数を呼ばないようにし、warningを出す
```